content/LCDC.md

```diff
@@ -94,14 +94,14 @@ Investigation needed.)
 
 LCDC.0 has different meanings depending on Game Boy type and Mode:
 
-### DMG, SGB and CGB in Non-CGB Mode: BG and Window Display
+### Non-CGB Mode (DMG, SGB and CGB in Non-CGB Mode): BG and Window Display
```

Comment:
Pinging @avivace for this: I think it's a good idea, but we should apply it everywhere else as well.

Reply:
What are you reffering to? DMG, SGB and CGB in Non-CGB Mode -> Non-CGB Mode?

Reply:
Maybe it's clearer now

Reply:
It certainly is. @avivace, do you think we should move this definition of "Non-CGB mode" to a glossary near the top of the document, so that it's explained for the bunch of places where it's used in the document?